src/workflows/demo_pipeline/main.nf

```diff
@@ -0,0 +1,47 @@
+workflow run_wf {
+  take:
+    input_ch
+
+  main:
+
+    output_ch =
+
+      input_ch
+
+        // Turn the Channel event with a list of files
+        // into multiple Channel events with one file.
+        // This involves either expanding a globbing parameter
+        // or multiple input files seperated by a `;`.
+        | expand
```

Comment:
can we omit this to make the workflow easier to understand?

Reply:
It does not appear explicitly in the quickstart nor the 'create-a-pipeline', so I would rather leave it in.